example/lib/pages/follow_my_location.dart

```diff
@@ -0,0 +1,370 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_map/flutter_map.dart';
+import '../widgets/drawer.dart';
+import 'package:latlong/latlong.dart';
+import 'dart:math' as math;
+import 'package:location/location.dart';
+import 'package:flutter/services.dart';
+
+//////////////////////////////
+// To set emulator location //
+//////////////////////////////
+// goto terminal
+// > adb devices
+//
+// telnet to device using the port associated with the device listing from the previous command
+// > telnet localhost <port>
```

Comment:
Can we remove this example? If you like, I can link to this example in the README. It's a little complicated to run for folks just wanted to see some use-cases.

There's probably a better way to demonstrate this with some fake interactions using streams: https://dart.dev/articles/libraries/creating-streams

Reply:
That's fair, I'll remove the example and if i get a chance at some point i'll put together a cleaner one. The main thing is having the isUserGesture distinction. 

Thank you for the feedback. :)

Reply:
Removed and rebased.